src/test/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogBuildListenerIT.java

```diff
@@ -147,7 +147,7 @@ public void testTraces() throws Exception {
         assertEquals("success", meta.get(CITags.STATUS));
         assertNotNull(meta.get(CITags.NODE_NAME));
         assertNotNull(meta.get(CITags.NODE_LABELS));
-        assertNull(meta.get(CITags._DD_HOSTNAME));
```

Comment:
@sarah-witt I think we cannot change this. The hostname must be "null".

Reply:
It's been resolved in this PR: https://github.com/jenkinsci/datadog-plugin/pull/288